src/buttons/CopyToClipboard/CopyToClipboard.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+
+import CSS from './styles.css';
+import copyToClipboard from '../../utils/copy-to-clipboard';
+
+const CopyToClipboard = ({ dataTestId, toCopyValue, toCopyLabel, copyingLabel, copiedLabel, toCopyClass, copyingClass, copiedClass, copyingFeedbackTime, copiedFeedbackTime }) => {
+	const copyStatusLookup = {
+		toCopy: { 
+		  class: toCopyClass,
+		  label: toCopyLabel,
+		},
+		copied: {
+		  class: copiedClass,
+		  label: copiedLabel,
+		},
+		copying: {
+		  class: copyingClass,
+		  label: copyingLabel,
+		},
+	  };
+
+	const copyTimeout = copyingFeedbackTime;
+	const copiedTimeout = copiedFeedbackTime;
+
+	const [copyStatus, setCopyStatus] = useState(copyStatusLookup.toCopy);
+
+	const afterCopy = () => {
+		setCopyStatus(copyStatusLookup.copied);
+		setTimeout(() => {
+			setCopyStatus(copyStatusLookup.toCopy);
+		}, copiedTimeout);
+	};
+
+	const handleOnClick = textToCopy => {
+		setCopyStatus(copyStatusLookup.copying);
+		setTimeout(() => {
+			copyToClipboard(textToCopy, afterCopy);
+		}, copyTimeout);
+	};
+
+	return (
+		<button data-testid={`${dataTestId}-button`} className={`${CSS.clipboard} ${CSS[copyStatus.class]}`} onClick={() => handleOnClick(toCopyValue)}>
```

Comment:
(nit) the `onClick` arrow-function is missing curly braces around its body (and a semicolon), please add them.